bee-message/src/output/signature_locked_asset_allowance.rs

```diff
@@ -0,0 +1,182 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{address::Address, payload::PAYLOAD_LENGTH_MAX, MessagePackError, MessageUnpackError, ValidationError};
+
+use bee_packable::{
+    error::{PackPrefixError, UnpackPrefixError},
+    PackError, Packable, Packer, UnknownTagError, UnpackError, Unpacker, VecPrefix,
+};
+
+use alloc::vec::Vec;
+use core::{
+    convert::{Infallible, TryInto},
+    fmt,
+};
+
+const ASSET_ID_LENGTH: usize = 32;
+
+/// No `Vec` max length specified, so use `PAYLOAD_LENGTH_MAX` / length of `AddressBalance`.
+const PREFIXED_BALANCES_LENGTH_MAX: usize = PAYLOAD_LENGTH_MAX / (ASSET_ID_LENGTH + core::mem::size_of::<u64>());
+
+/// Error encountered packing a `SignatureLockedAssetAllowanceOutput`.
```

Comment:
No `Allowance`

Reply:
Also in the file name

Reply:
Hmm, must have not made it through the rebasing 🤔 Will change.